src/java_1/collections/CollectionsChallenges.java

```diff
@@ -1,61 +1,125 @@
-package java_1.collections;
-
 import java.util.*;
 
 public class CollectionsChallenges {
 
 	public static void main(String[] args) {
-//		Complete the code challenges for the following static methods
+//		Complete the code challenges for the following static methods I forgot to make a branch
+
+		ArrayList<Integer> arrayList = new ArrayList<>(
+				Arrays.asList(12, 41, 435, 76, 43, 12, 10, 34)
+		);
+		int[] array = {1,2,3,4,5,6,7};
+		ArrayList<String> arrayList1 = new ArrayList<String>();
+		arrayList1.add("hello1");
+		arrayList1.add("hello1");
+		arrayList1.add("hello1");
+		arrayList1.add("hello2");
+		arrayList1.add("hello3");
+		arrayList1.add("hello4");
+		arrayList1.add("hello5");
+		arrayList1.add("hello5");
+		arrayList1.add("hello4");
+		arrayList1.add("hello3");
+
+		Person anthony = new Person("Anthony",31);
+		Person anthonyClone = new Person("Anthony Clone",31);
+		Person yesica = new Person("Yesica",33);
+		Person yesicaClone = new Person("Yesica Clone",33);
+		Person eddison = new Person("Eddison",2);
+		Person grumps = new Person("Grumps",61);
+
+		Person [] personArr = {anthonyClone,anthony,yesica,eddison,grumps,yesicaClone};
+
+		System.out.println(covidVaccineQueue(personArr));
+
+
 	}
 
-	int findVal(int val, ArrayList<Integer> arrList) {
-//		find a value in the input array list.  return it if found.  If not, return 0.
-		return 0;
+	static int findVal(int val, ArrayList<Integer> arrList) {
```

Comment:
Some example output would really help for this and other methods in this exercise.